meshroom/ui/qml/Viewer/HdrImageToolbar.qml

```diff
@@ -23,9 +23,13 @@ FloatingPane {
     property real gammaValue: Math.pow(gammaCtrl.value, slidersPowerValue)
     property alias channelModeValue: channelsCtrl.value
     property variant colorRGBA: null
+    property variant mousePosition: {return {x:0, y:0} }
```

Comment:
```suggestion
    property variant mousePosition: ({x: 0, y: 0})
```
The initialization of `x` and `y` remains the same, but this is more QML-like (as a reference: https://doc.qt.io/qt-6.5/qml-var.html#property-value-initialization-semantics)